docs/zkapps/tutorials/04-zkapp-ui-with-react.mdx

```diff
@@ -619,14 +670,43 @@ Before you can deploy your project to GitHub Pages, you must push it to a new Gi
 - In this tutorial, the project name is `04-zkapp-browser-ui`.
 - The `zk project` command created the correct project name strings for `04-zkapp-browser-ui` in the `next.config.js` and `src/pages/reactCOIServiceWorker.ts` files. 
 
-To deploy the UI, run the `npm run deploy` command in your local `/04-zkapp-browser-ui/ui/` directory. 
+To configure your project to deploy to GitHub Pages:
+
+1. Edit the `/04-zkapp-browser-ui/ui/ghp-postbuild.js` file.
+1. Add your repository name. For this tutorial, use `04-zkapp-browser-ui`:
```

Comment:
I updated the CLI to add the repository name to this file `/04-zkapp-browser-ui/ui/ghp-postbuild.js`.  We can either remove this section or mention to update the repoURL if the repo name is different from the project name when a user enters `zk project <project name>`.  In this case the project name is `04-zkapp-browser-ui` which is the same as the repo name.